app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java

```diff
@@ -369,9 +382,21 @@ public boolean onQueryTextChange(String newText) {
 				Collections.sort(cafeList);
 
 			} else {
-				String json = NetworkUtilities.getJSON();
-				dbHelper.addData(json);
-				cafeList = JsonUtilities.parseJson(json, getApplicationContext());
+                if (JSON_FALLBACK) {
+                    String json = NetworkUtilities.getJSON();
+                    dbHelper.addData(json);
+                    cafeList = JsonUtilities.parseJson(json, getApplicationContext());
+                } else {
+                    while (!NetworkUtilities.EATERIES_LOADED) {
+                        try {
+                            Thread.sleep(10);
+                        } catch (InterruptedException e) {
+                            e.printStackTrace();
+                        }
+                    }
+                    List<AllEateriesQuery.Eatery> eateries = NetworkUtilities.getEateries();
+                    cafeList = JsonUtilities.parseEateries(eateries, getApplicationContext());
+                }
```

Comment:
this indentation also seems off, between this line & the next there's a huge difference

Reply:
I see no issues in Android Studio, so I believe this is the linter causing issues.